etc/development.ini.in

```diff
@@ -42,6 +42,14 @@ adhocracy.abuse_handler_mail = abuse_handler@unconfigured.domain
 # Template for the subjects of messages sent to users (Python format string,
 # usable variables: {site_name}, {sender_name}, {title})
 adhocracy.message_user_subject = [{site_name}] Message from {sender_name}: {title}
+# Only accept registration requests with valid captcha solutions
+adhocracy.captcha.required = ${thentos_captcha:required}
+# Where the frontend sends captcha traffic
+adhocracy.captcha.backend_url = ${thentos_captcha:backend_url}
+# Where the backend sends captcha traffic (in development mode, this
+# is the same as captcha backend; if nginx is dispatching inbound
+# requests, this is just the frontend url)
+adhocracy.captcha.frontend_url = ${thentos_captcha:frontend_url}
 # If true, new user accounts are activated immediately without email verification
```

Comment:
I would a the thentos namespace, to make clear that this is related to thentos, like adhocracy.thentos_captcha_frontend_url